script/Ciel/Day 14/14_01B_01_CIEL14B_1E.txt

```diff
@@ -321,122 +346,131 @@ I hate Roa. I'll kill Roa.
 [sha:d271023bdad2d581c3153d374a77d0f1f6dfdc51]{
 -- Page 233, Offset 39034.
 -- ■■が好き。■■が欲しい。
-I love ■■. I want ■■.
+I love ■■■. I want ■■■.
 }
 [sha:cc17db55fe7d0ee9fb4928625abe891b03eda10a]{
 -- Page 233, Offset 39035.
 -- ロアが憎い。ロアを殺す。
-   I hate Roa. I'll kill Roa.
+I hate Roa. I'll kill Roa.
 }
 [sha:0d5d7d8012536be51059d484f5e21692b473a87e]{
 -- Page 233, Offset 39036.
 -- 志貴が好き。志貴が欲しい。
-   I love Shiki. I want Shiki.
+I love Shiki. I want Shiki.
 }
 [sha:00192d5c726088f260f1a73fcc2444203dcdadb4]{
 -- Page 233, Offset 39037.
 -- 志貴が憎い。志貴が欲しい。
-      I hate Shiki. I'll kill Shiki.
+I hate Shiki. I want Shiki.
 }
 [sha:f7c9fdf1b47f907411c40a81d9e63a9b9c03c2af]{
 -- Page 233, Offset 39038.
 -- 志貴がきらい。だいっきらい……！
-      I hate Shiki. I hate him so much...!
+I hate Shiki. I hate him so much!
 }
 [sha:91c46cb67134592cb4b1fae6422e00ed84dd249b]{
 -- Page 234, Offset 39039.
 -- 思考が乱れる。
-Her thoughts run amok.
+Her thoughts are running wild.
 }
 [sha:6a6bbf97db34ac9f25d3a1a10169d1ea9e83b2ca]{
 -- Page 234, Offset 39040.
 -- 整理が付かない。
-She can't put it in order.
+She can't make sense of them all.
 }
 [sha:6bed4a8e9e271eaadb088d550add64f966b244df]{
 -- Page 234, Offset 39041.
 -- 意識すればするほど順番が変わってしまう。
-The more she tries to make sense of it, the more disorderly she becomes.
+The more she tries to think,
+ the more muddled her mind becomes.
 }
 [sha:629585d7f9e6671f51537bbb39037a2da939690d]{
 -- Page 235, Offset 39042.
 -- なので、彼女は考える事はやめる事にした。
-As such, she decided to stop thinking.
+And so, she decides to stop thinking.
 }
 [sha:07d6175143ebd3db43edda9dd8a821055b33063f]{
 -- Page 235, Offset 39043.
 -- 自分の気持ちを考えたり、彼の気持ちを考えたりするから混乱する。
-Things are chaotic because she continues to think about her own feelings and about him.
+She's confused because she's trying to think about her emotions; about his emotions.
 }
 [sha:1f25ea6d796e25cb43eb9335ac1b596008ef8279]{
 -- Page 235, Offset 39044.
 -- 彼女は原始的に、それ以上に純粋無垢に、<や|・><り|・><た|・><い|・><事|・><だ|・><け|・><を|・><や|・><る|・>獣になった。
-She turns herself into a primitive beast; so primitive, she chooses only to do the things she wants to do.
+So instead, she turns herself into the platonic ideal of a primitive beast――one driven by nothing but its own desires.
```

Comment:
I personally hate this way of using "platonic ideal" but that may just be the prescriptivist inside of me.
It's sorta not... proper? I guess. It's a very recent sense of the word.